index.html

```diff
@@ -28,19 +28,192 @@ <h1 class="introduction">I'm Emmanuel, glad to see you</h1>
+        </figure>
+        <div class="works-description">
+            <h3 class="my-works-project-title">Multi-Post
+                Stories</h3>
+            <ul class="works-headers">
+                <li class="title-header">CANOPY</li>
+                <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+                <li class="title-header-general">Back End Dev</li>
+                <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+                <li class="title-header-general">2015</li>
+            </ul>
+            <p class="project-description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+        </div>
+        <section class="project-languages">
+            <div class="coding-languages">html</div>
+            <div class="coding-languages">css</div>
+            <div class="coding-languages">javascript</div>
+        </section>
+        <section class="project-more">
+            <button class="works-see-more"><span class="more-text">See Project</span></button>
+        </section>  
+    </div>
+    <!--My projects end//-->
+
+    <!-- About Myself Start-->
+    <section class="headline">
+        <h1 class="introduction">About Myself</h1>
 
+        <p class="description">I’m a software developer! I can help you build a product , feature or website Look 
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t 
+            hesitate to contact me.</p>    
+        <h4 class="connect">LETS CONNECT</h4>
         <ul class="social">
             <li><a href="#"><img src="images/twit.png" alt="Twitter"></a></li>
             <li><a href="#"><img src="images/Vector.png" alt="Linkedin"><img src="images/in.png" alt=""></a></li>
             <li><a href="#"><img src="images/m.png" alt="Mi"></a></li>
             <li><a href="#"><img src="images/cat.png" alt="Guthub"></a></li>
             <li><a href="#"><img src="images/peace.png" alt="Alison"></a></li>
         </ul>
+        <section class="project-more" style="margin-left: 14px;">
+            <button class="works-see-more"><span class="more-text">Get my resume</span></button>
+        </section> 
+        <section class="coding-languages-i-know">
+            <div class="my-skills">Languages</div>
+            <div><i class="fa fa-angle-down fa-lg"></i></div>
+        </section>
+        <section class="actual-languages">
+            <ul class="language-list">
+                <li class="programing-languages">
+                    <img src="images/js.png" alt="Javascript" class="language-icons">
+                    <h3 class="align-language-text">Javascript</h6> 
+                </li>
+                <li class="programing-languages">
+                    <img src="images/html.png" alt="HTML" class="language-icons">
```

Comment:
Kindly modify the languages section to match your chosen design. Consider the alignment and margin of the drop-downs. Please, also adjust the background image in such a way as not to appear behind the skills arrow.

Your design:
![image](https://user-images.githubusercontent.com/20554583/192076672-864bb173-e536-483b-9581-45e9f4632938.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/192076808-0f77b873-b105-46e9-9513-7a260737fd3e.png)